Dockerfile

```diff
@@ -28,17 +25,14 @@ WORKDIR /app/${package}
 
 # Run tests and return coverage analysis
 RUN python -m coverage run tests/test_${package}.py \
- && python -m coverage html -d /htmlcov --omit=/usr/local/*
+&& python -m coverage html -d /htmlcov --omit=/usr/local/*
 
 
 # STAGE 2 -- Build final plugin image
-FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-osbase:0.3.1@sha256:0e9384416ad5dd8810c410a87c283ca29a368fc85592378b85261fce5f9ecbeb
+FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-osbase:stream9
 ARG package
-RUN dnf -y install pcp pcp-export-pcp2json pcp-system-tools
-
+RUN dnf -y install pcp pcp-export-pcp2json pcp-system-tools procps-ng util-linux-core; useradd -U pcp
```

Comment:
Use `&&` instead of `;`?

(Or, consider running `useradd` in a separate `RUN` command, possibly before the `dnf`.)